src/scenarios/staticconsoletemplate/test.py

```diff
@@ -11,6 +11,7 @@
                         exename=EXENAME, 
                         startupmetric='TimeToMain',
                         startup=True,
+                        sdk=True,
```

Comment:
Will this now also be true for when we run the staticconsoleempty tests not eh SDK tests?

Reply:
resolved offline :)